DaS-PC-MPChan/DSCM.vb

```diff
@@ -526,24 +526,34 @@ Public Class DSCM
                     tmpWorld = tmpFields(5)
                     tmpUpdMinute = TimeOfDay.TimeOfDay.Minutes
 
-                    If tmpSteamID.Length = 16 Then
-
-
-                        'Autoconnect to any nodes when Currnodes are under 4
-                        If Val(txtCurrNodes.Text) < 4 Then
-                            attemptConnSteamID(tmpSteamID)
+                    'Calculations for the IRC node-connect feature.
+                    Dim CurrNodesThreshold As Integer = 4
```

Comment:
> how to accept part of a submission
On the conversation tab of the PR, at the very bottom click `command line instructions`. Follow step 1. You should now have a local branch that contains the changes. The first token after `git checkout -b` is name of that branch.

Now do 
```
git checkout master
git merge ***branchname*** --squash
git gui
```
You can now selective stage and unstage lines. Stage the parts you want, unstage everything else.
![image](http://i.stack.imgur.com/tfKNE.png)
Commit.
If you want to attribute the commit to hydra-chan, do `git commit --amend --author="Author Name <email@address.com>"`
Do `git reset --hard` to discard all the changes you didn't commit.
You can delete the left-over branch with `git branch -D **branchname**`